### PR TITLE
feature-clip-board-manager: Add new method to copy content to clipboard

### PR DESCRIPTION
**Pull request by paskahdawenan**

### CODE REVIEW

File: extensions/src/main/java/com/tunaikumobile/extensions/ContextExtensions.kt [hunk 1]
```diff
@@ -59,4 +62,15 @@ fun Context.isNetworkConnected(): Boolean {
         this.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
     val activeNetworkInfo = connectivityManager.activeNetworkInfo
     return activeNetworkInfo != null && activeNetworkInfo.isConnected
+}
+
+fun Context.copyToClipboard(label: String, content: String) {
+    val clipBoardManager =
+        this.getSystemService(CLIPBOARD_SERVICE) as ClipboardManager
```
Comment (nmaingkolang) [inline extensions/src/main/java/com/tunaikumobile/extensions/ContextExtensions.kt:69]: langsung casting aman kanda @paskahdawenan?

Comment (paskahdawenan) [reply extensions/src/main/java/com/tunaikumobile/extensions/ContextExtensions.kt:69]: Aman kanda sudah ditest kok

File: extensions/src/main/java/com/tunaikumobile/extensions/StringExtensions.kt [hunk 2]
```diff
@@ -157,4 +165,15 @@ fun String.isInteger(): Boolean = this.toIntOrNull() != null
  */
 fun String.removeNumberDelimiter(): String {
     return this.replace(".", "")
+}
+
+fun Context.copyToClipboard(label: String, content: String) {
```
Comment (Suyanwar) [inline extensions/src/main/java/com/tunaikumobile/extensions/StringExtensions.kt:170]: Harusnya gini kanda
String.copyToClipBoard(context: Context, label: String) {
   ...
}

Comment (paskahdawenan) [reply extensions/src/main/java/com/tunaikumobile/extensions/StringExtensions.kt:170]: Iyak kanda. Ini udeh gw ubah wkwkkwk

**Review by Suyanwar — APPROVED** [review]

**Pull request merged**